mathics/core/expression.py

```diff
@@ -243,7 +243,8 @@ def do_format(self, evaluation, form):
             include_form = False
             if head in formats and len(self.get_leaves()) == 1:
                 expr = self.leaves[0]
-                if not (form == 'System`OutputForm' and head == 'System`StandardForm'):
+                # removing 'True' below breaks the 'RGBColor[1,0,0]//StandardForm' test cases.
+                if True or not (form == 'System`OutputForm' and head == 'System`StandardForm'):
```

Comment:
Thinking about this a bit more, does Mathematica show "`-Graphics-`" when you do `StandardForm[RGBColor[1, 0, 0]]` in its console interface?

I think if we're just doing this for the test, another way to test this might be to use `ToBoxes[RGBColor[1, 0, 0]]` instead of `StandardForm`. If you do this the test would also fail if the box form changes.